osquery/tests/integration/tables/last.cpp

```diff
@@ -16,28 +15,28 @@
 
 namespace osquery {
 
-class last : public IntegrationTableTest {};
-
-TEST_F(last, test_sanity) {
-  // 1. Query data
-  // QueryData data = execute_query("select * from last");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See IntegrationTableTest.cpp for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidatatioMap row_map = {
-  //      {"username", NormalType}
-  //      {"tty", NormalType}
-  //      {"pid", IntType}
-  //      {"type", IntType}
-  //      {"time", IntType}
-  //      {"host", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+class Last : public IntegrationTableTest {};
```

Comment:
The name `Last` is pretty popular as far as all test are linked in one binary it would be safer to have a more complex name for this class, or just hide all this testing code in the unnamed namespace.